open-company/open-company-auth
# Jwt slack auth

card: https://trello.com/c/Ee4lE6Dd

review with this:
https://github.com/open-company/open-company-web/pull/24

to test:
- deploy this branch on staging (use auth-service branch of infrastructure)
- deploy [user-auth](https://github.com/open-company/open-company-web/pull/24) branch of open-company-web too
- open this page [https://staging.opencompany.io/login](https://staging.opencompany.io/login) on your browser
- tap on the slack button
- after you login with slack you should be redirected to https://staging.opencompany.io/companies
- check the cookies, you should have one called `jwt` containing you token
- check the last requests to the API server, all these should contains a `Authorization` cookie containing a value like `Bearer {you JWT token}`

tests are TBD
